amazon-kinesis-client/src/test/java/software/amazon/kinesis/config/BasicReleaseCanaryConfig.java

```diff
@@ -0,0 +1,108 @@
+package software.amazon.kinesis.config;
+
+import lombok.extern.slf4j.Slf4j;
+import software.amazon.awssdk.http.Protocol;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.kinesis.common.InitialPositionInStream;
+import software.amazon.kinesis.common.InitialPositionInStreamExtended;
+import software.amazon.kinesis.retrieval.RetrievalConfig;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+/**
+ * Basic config for a release canary (streaming) with default settings
+ */
+@Slf4j
+public class BasicReleaseCanaryConfig implements KCLAppConfig {
+    @Override
+    public String getStreamName() {
+        return "";
+    }
+
+    @Override
+    public int getShardCount() {
+        return 10;
+    }
+
+    @Override
+    public String getApplicationName() {
+        return "";
+    }
+
+    @Override
+    public String getEndpoint() {
+        return "";
+    }
+
+    @Override
+    public Region getRegion() {
+        return Region.US_WEST_2;
+    }
+
+
+    /**
+     * This will get the credentials that are provided in the maven command
+     * when running integration tests if any are provided through -Dcredentials=iamUser
+     * Otherwise, iamUser will be null and the test will search for default credentials
+     * in the test environment.
+     */
+    @Override
+    public String getProfile() {
+        String iamUser = System.getProperty("credentials");
```

Comment:
If this argument is provided, a [ProfileCredentialsProvider](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/profile/ProfileCredentialsProvider.html) is created. Otherwise, a default AWSCredentialsProvider is created to the account whos credentials are found at .aws/configuration as mentioned in the README.